packages/forms-web-app/src/sass/expand-collapse-comments.scss

```diff
@@ -0,0 +1,12 @@
+.hidden {
+    display: none;
+}
+
+.visible {
+    display: inline;
+}
+
+.expand-collapse-button {
+    color: #1D70B8;
+    cursor: pointer;
```

Comment:
When i was testing what was working using <a> without the href doesnt automatically make the cursor a pointer. However when adding href it reloads the page or wherever its linked to. Having an empty string reloads the page and doesnt leave the comment expanded, having a # reloads the page and does leave the comment expanded. 

Can use javascript to stop the default behaviour of the page reloading as it goes back to the top of the page otherwise. 

Happy to remove the cursor from here and add the href and js to stop the default, not sure which way is preferable?

Reply:
ah ok, my mistake can leave as is